app/_hub/kong-inc/jwt-signer/overview/manage-keys.md

```diff
@@ -0,0 +1,59 @@
+---
+nav_title: Manage key signing
```

Comment:
@lena-larionova I was wondering what your thoughts are on where I decided to put this information. I thought about maybe making another page in the "using this plugin" section, but those seemed specific to config files, and this contained more information than just config info. 

Also, did I miss anything like a nav when I added this? I followed the plugin docs instructions and didn't see anything about a nav, so I figured things in this folder were auto added to the nav.

Reply:
Oh, I'm having Veena answer some questions, so this isn't quite ready for a copyedit yet, I just wanted to get your opinion on the formatting/if I did the overall structure correctly.

Reply:
Ideally we want this under how-to. Only reason it's currently specific to config files is because I haven't gotten to cleaning up this plugin yet :)

Reply:
Here's an example of a plugin that does that: https://github.com/Kong/docs.konghq.com/tree/main/app/_hub/kong-inc/acme/how-to

You'll notice it sets `nav_title` in the front matter as well.